core/platform/email/mailgun_email_services.py

```diff
@@ -39,6 +39,8 @@ def send_mail(
       - html_body: str. The HTML body of the email. Must fit in a datastore
           entity.
       - bcc_admin: bool. Whether to bcc feconf.ADMIN_EMAIL_ADDRESS on the email.
+      - reply_to_id: str or None. The unique reply-to id used in reply-to email
+          sent to recipient.
```

Comment:
Indent everything by 4 spaces.

Reply:
I just added a missing argument.
The indentation is exactly similar to all files across the project.

Reply:
It's because it was old way of writing. Now we are following indentation by 4 spaces only. Please follow the latest guidelines.

Reply:
Done